doc/screenshot.js

```diff
@@ -3,15 +3,13 @@ import puppeteer from 'puppeteer';
 export const screenshot = async (html) => {
   let timeoutCount = 0;
   const timeoutLimit = 3;
-  const pngDataList = [];
   const imageUrlList = [];
   for (;;) {
     let browser;
     try {
-      pngDataList.length = 0;
       imageUrlList.length = 0;
 
-      const browser = await puppeteer.launch();
+      const browser = await puppeteer.launch({ headless: true });
```

Comment:
Can puppeteer run in a head having mode?

Reply:
It seems so.